crates/ruff_python_stdlib/src/str.rs

```diff
@@ -11,9 +11,6 @@ pub const SINGLE_QUOTE_PREFIXES: &[&str] = &[
 pub const TRIPLE_QUOTE_SUFFIXES: &[&str] = &["\"\"\"", "'''"];
 pub const SINGLE_QUOTE_SUFFIXES: &[&str] = &["\"", "'"];
 
-pub static STRING_QUOTE_PREFIX_REGEX: Lazy<Regex> =
-    Lazy::new(|| Regex::new(r#"^(?i)[urb]*['"](?P<raw>.*)['"]$"#).unwrap());
```

Comment:
(Drive-by fix -- this is no longer used anywhere, but unrelated to the change at hand.)

Reply:
I noticed that we use `pub` in many places where it isn't necessary. It would be good in my view, to be as restrictive as possible when it comes to defining visibility. Not using `pub` has the advantage that Rust flags unused symbols.